Cargo.toml

```diff
@@ -93,9 +96,11 @@ std_instead_of_core = { level = "allow", priority = 127 }              # we shou
 string_add = { level = "allow", priority = 127 }                       # simple concat ok
 string_lit_chars_any = { level = "allow", priority = 127 }             # favor readability until a perf case comes up
 todo = { level = "warn", priority = 127 }                              # warn todos
+verbose_file_reads = { level = "allow", priority = 127 }
```

Comment:
Yeah this was an issue with fs4, I removed it because it was complaining that
``` rust
let file = fs::OpenOptions::new().read(true).open("foo.txt");
file.exclusive_lock()
```

should be
```
fs::read(<file>)
```

The issue is that, fs4 requires verbose file reads, and we will most likely run into it later when we start to use that. But for not I can remove it